From cd13e478935eac5512185a7532afb725e45373b1 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 14 Apr 2017 15:36:14 +0800
Subject: [PATCH] arm: dts: rk3288-evb-rk818: set tsadc default pinctrl to gpio

When rockchip,hw-tshut-mode is 0, means that tshut mode is CRU,
then the pinctrl should default to be GPIO.

If the tshut mode is GPIO, then the pinctrl should default to be
OTP_OUT.

Change-Id: Ic6e53a96823baf4671f9bad261ed34586512634f
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-rk818.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3288-evb-rk818.dts b/arch/arm/boot/dts/rk3288-evb-rk818.dts
index 0ac1fe591dec..12e54c22f041 100644
--- a/arch/arm/boot/dts/rk3288-evb-rk818.dts
+++ b/arch/arm/boot/dts/rk3288-evb-rk818.dts
@@ -330,6 +330,7 @@
 &tsadc {
 	rockchip,hw-tshut-mode = <0>; /* tshut mode 0:CRU 1:GPIO */
 	rockchip,hw-tshut-polarity = <1>; /* tshut polarity 0:LOW 1:HIGH */
+	pinctrl-1 = <&otp_gpio>;
 };
 
 &pinctrl {
-- 
2.35.3

